### DISCUSSION TITLE
Added static Basic Authentication policy but no pop up

### DISCUSSION THREAD

**Discussion by AntonOttoW** [apiman/apiman#1452] [Help] [answered]
So I followed the crash course and added the Basic Authentication policy with a static user for testing purposes. When accessing the api endpoint with the allocated APIKey, no popup appears to capture the username and password and it just returns Unauthorized immediately.

When I try to access the same url via Postman and add a Basic Auth header I just get 401 Unauthorized back.

What am I missing from a configuration point of view. The gateway on manager tests successfully, so connection there seems good.

**Comment by msavy** [comment, marked as answer]
Good morning!

Can you post a screenshot of your basic auth policy config and a `curl -v` or similar through your gateway that you were expecting to have BASIC auth enabled on?

**Reply by AntonOttoW** [reply]
See screenshots below. I just want to mention, when I create a Public API with No Policies, I get the Unauthorized error, similar to the Postman screenshot below here. When I specify apimanager/apiman123! as Basic auth then it comes back with an error "RESTEASY003210: Could not find resource for full path"

![BASICAuthConfig](https://user-images.githubusercontent.com/28915136/129919199-4863f978-31fc-4ad0-96e5-509a6782b5dd.PNG)
![PostmanCall](https://user-images.githubusercontent.com/28915136/129919192-889b6bcd-7d81-4e8d-836a-de01dbf432cd.PNG)
![GatewayConfigTest](https://user-images.githubusercontent.com/28915136/129919202-d38c9da4-6f5a-4d90-91a8-39a84ca9e51e.PNG)

**Reply by msavy** [reply]
The Apiman Gateway API endpoint is where Apiman Manager pushes configuration to tell the Apiman Gateway how to route traffic and which policies to apply. You are not supposed to invoke that endpoint to proxy traffic.

Assuming you are using a Public API: once you have published your API, you go to the 'endpoint' tab on your API and see the correct URL to use.

![image](https://user-images.githubusercontent.com/423513/129923301-ccd23e14-df46-49c2-9afa-c2c3b940a2a2.png)

**Reply by AntonOttoW** [reply]
Thank you so much. I misconfigured the api endpoint in apiman.properties to apiman-gateway-api.... changing it to apiman-gateway and now it's working!!
